openapi-v1.yaml

```
@@ -339,3 +473,68 @@ components:
           x-go-name: Subject
           type: string
           description: OAuth 2.0 Subject for the user
+
+    CreateGroup:
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          description: a name for the group
+        description:
+          type: string
+          description: a description for the group
+
+    UpdateGroup:
+      properties:
+        name:
+          type: string
+          description: a name for the group
+        description:
+          type: string
+          description: a description for the group
+
+    Group:
+      required:
+        - id
+        - name
+        - owner
+        - members
+      properties:
+        id:
+          x-go-name: ID
+          type: string
+          x-go-type: gidx.PrefixedID
+          description: ID of the group
+        name:
+          type: string
+          description: a name for the group
+        description:
+          type: string
+          description: a description for the group
+        owner:
+          type: string
+          x-go-type: gidx.PrefixedID
+          description: ID of the owner of the group
```

Should this be owner_id? Also, looking at some of the other resource types, we don't seem to typically return this field.
```suggestion
        owner_id:
          x-go-name: OwnerID
          type: string
          x-go-type: gidx.PrefixedID
          description: ID of the owner of the group
```